mesh_segmenter/CMakeLists.txt

```diff
@@ -4,27 +4,61 @@ project(mesh_segmenter)
 find_package(VTK 7.1 REQUIRED NO_MODULE)
 include(${VTK_USE_FILE})
 
-find_package(catkin REQUIRED
+find_package(catkin REQUIRED COMPONENTS
+  vtk_viewer
```

Comment:
vtk_viewer isn't actually needed for this package. It will be used in the tests only in #22. It's not used at all right now

Reply:
Changed it to a test-depend.